tests/test_log_utils.py

```
@@ -12,6 +12,21 @@
 pytestmark = [pytest.mark.unit, pytest.mark.infrastructure]
 
 
+def _app_handlers():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_app_handlers`

(ANN202)